CNAME

```diff
@@ -1 +0,0 @@
-cephalopod.equipment
```

Comment:
GitHub sometimes requires this file so the website can be properly displayed on its own domain. We couldn't really figure out when or why but when t was missing, the infra setup reverted to something.github.io so if there's no other reason to delete the file, let's keep it.